README.md

```diff
@@ -22,35 +22,39 @@ $ npm i storyblok -g
 
 ## Commands
 
-
 ### login
 
 Login to the Storyblok cli
 
 ```sh
 $ storyblok login
 ```
+
 #### Login options
 
 ##### Options for Login with email and password
-* `email`: your user's email address
-* `password`: your user's password
+
```

Comment:
Please revert the changes in the readme, this `*` is here to show the user the required params for each command.